model/docs/getting_started.qmd

```diff
@@ -33,340 +57,3 @@ class MyRandVar(RandomVariable):
 Thus, when a `Model` samples from `MyRandVar`, it could be either adding random variables to be estimated (first case) or just retrieving some quantity needed for other calculations (second case.)
 
 The `Model` metaclass provides basic functionality for estimating and simulation. Like `RandomVariable`, the `Model` metaclass has a `sample()` method that defines the model structure. Ultimately, models can be nested (or inherited), providing a straightforward way to add layers of complexity.
```

Comment:
Maybe after this, we can point to one of our models or tutorials as an example / "next step?"